run_runtime.go

```diff
@@ -54,3 +63,46 @@ func generateRuntimeCOpts(clicontext *cli.Context) ([]containerd.NewContainerOpt
 	o := containerd.WithRuntime(runtime, runtimeOpts)
 	return []containerd.NewContainerOpts{o}, nil
 }
+
+// WithSysctls sets the provided sysctls onto the spec
+func WithSysctls(sysctls map[string]string) oci.SpecOpts {
+	return func(ctx context.Context, client oci.Client, c *containers.Container, s *runtimespec.Spec) error {
+		if s.Linux == nil {
+			s.Linux = &runtimespec.Linux{}
+		}
+		if s.Linux.Sysctl == nil {
+			s.Linux.Sysctl = make(map[string]string)
+		}
+		for k, v := range sysctls {
+			s.Linux.Sysctl[k] = v
+		}
+		return nil
+	}
+}
+
+func WithResources(resources *LinuxContainerResources) oci.SpecOpts {
```

Comment:
Should be around here: https://github.com/containerd/nerdctl/blob/master/run_cgroup.go#L55-L61